harness/determined/common/experimental/user.py

```diff
@@ -0,0 +1,88 @@
+from typing import Optional
+
+from determined.common import api
+from determined.common.api import bindings
+
+
+class User:
+    def __init__(
+        self,
+        user_id: int,
+        username: Optional[str],
+        admin: Optional[bool],
+        session: api.Session,
+        active: Optional[bool] = True,
+        display_name: Optional[str] = None,
+        agent_uid: Optional[int] = None,
+        agent_gid: Optional[int] = None,
+        agent_user: Optional[str] = None,
+        agent_group: Optional[str] = None,
+    ):
+        self.username = username
+        self.admin = admin
+        self.user_id = user_id
+        self.active = active
+        self.agent_uid = agent_uid
+        self.agent_gid = agent_gid
+        self.agent_user = agent_user
+        self.agent_group = agent_group
+        self._session = session
+        self.display_name = display_name
+
+    def reload(self, raw: Optional[bindings.v1User] = None) -> None:
```

Comment:
let's have it as private for now

```suggestion
    def _reload(self, raw: Optional[bindings.v1User] = None) -> None:
```